src/Service.Host/client/src/components/__tests__/parts/PartTemplateRender.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { cleanup, screen } from '@testing-library/react';
+import render from '../../../test-utils';
+import PartTemplateSearch from '../../parts/PartTemplatesSearch';
```

Comment:
convention is to name the test file the same as the componenet. So this file should be called PartTemplateSearch.test.js. 

Although I think in this case i'd rename your component since it isn't technically a search